examples/src/Example00SingleFieldQuery.elm

```diff
@@ -34,12 +35,12 @@ import Swapi.Scalar
 
 
 type alias Response =
-    String
+    Int
 
 
 query : SelectionSet Response RootQuery
 query =
-    Query.hello
+    Query.now |> SelectionSet.map (Time.toSecond Time.utc)
```

Comment:
For the purposes of showing this prototype, I changed this to reference `Query.now`. Since it's using a custom decoder, the type of `Query.now` is `SelectionSet Time.Posix RootQuery`.